gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Tue Aug 08 15:57:04 PDT 2017
```

Comment:
I'll rebase it once it's checked in so I don't have to keep rebasing while you're making changes.